### PR TITLE
{bio,numlib}[GCC/8.2.0-2.31.1] Beast v2.5.2, beagle-lib v3.1.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3106.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8af1ec80b715221abaa00280b59d89df for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2473.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f6d932882f4dd6914b20a7cdc8bcfd1a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2035.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/749b4a9a90b461ea25983343d8692781 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 4 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/56b81929809c2ff2c58a58f28cb1600d for a full test report.

**Comment by boegel** [comment]
@verdurin You're missing the fix added in https://github.com/easybuilders/easybuild-framework/pull/2848 (not a part of EasyBuild v3.9.0 yet)

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ba5e5818f104db6f8124d7db194743e8 for a full test report.

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-3.1.2-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+dependencies = [('Java', '11', '', True)]
+
+builddependencies = [('Autotools', '20180311')]
+
+preconfigopts = "./autogen.sh && "
+
+sanity_check_paths = {
+    'files': ["include/libhmsbeagle-1/libhmsbeagle/%s" % includefile
+              for includefile in ["beagle.h", "platform.h"]] +
+             ["lib/libhmsbeagle%s.so" % libfile
```
Comment (verdurin) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-3.1.2-GCC-8.2.0-2.31.1.eb]: @boegel - oughtn't this to be handled with `SHLIB_EXT`?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic32 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ec2ac346096311948fffc4cd792c92ba for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**